README.md

```diff
@@ -247,6 +247,18 @@ Some settings can be customized for the entirety of the deployment, they are:
 
 To customize any of these, copy `settings.yaml.example` to `settings.yaml` and add, remove or update the ones you wish to change'
 
+#### Customize Available Boxes
+
+The list of available boxes can be customized by setting an exclude list in `settings.yaml`. This allows faster `vagrant status` calls as well as reducing the the scope of boxes a user sees to tailor to their use cases. To specify boxes to exclude add the following to `settings.yaml`, for example, to remove fips, fedora and any Foreman 1.2X boxes from view:
+
+```
+boxes:
+  exclude:
+    - fips
+    - fedora
+    - '1.2'
+```
```

Comment:
@ehelms should we mention or add an example with regex here now that its an option?

Reply:
See, this is kind of an issue now IMO - this `'1.2'` exclude in the example is going to match a box I have named (i.e.) `centos7-152.example.com`. I don't think that's expected or desired, do you?

Reply:
its still possible to match "1.2" with "1\\.2"
```
2.6.3 :017 > "centos7-152.example.com".match(/#{"1\\.2"}/)
 => nil 
2.6.3 :018 > "centos1.2.example.com".match(/#{"1\\.2"}/)
 => #<MatchData "1.2"> 
```
but yes, we should mention that the expressions are treated as ruby-flavor regex and add examples. 

I would prefer to keep regex to be able to fine tune the exact boxes that should be ignored. My thought is to be able to specify exactly what you want without side effects, such as a range of versions or any box that starts with "foo" but allow boxes that include "foo" in the middle of the name. This does come with handling special characters but I think it could be documented and examples can be added.

Reply:
I'd lean towards the 'Keep it Simple' mantra. If this is a deal breaker for your use case, we could allow advanced users to specify a regex literal (`/^foo/`) if they want regex but keep the defaults easy to use and understand. After all, people tend to not read docs unless the system behaves in a non-intuitive way.

Reply:
I think it just comes down to preference. A user would have to read the docs to even know about the feature so I think either will would work fine with good documentation. 

@ehelms any thoughts?

Reply:
If you can provide an example regex I can add that and it would help to settle the discussion.

Reply:
@ehelms  Here is an example showing off simple to more specific matching:
```yaml
boxes:
  exclude:
    - "katello" # exclude any box containing "katello"
    - "ubuntu1604-foreman-1\\.24" # exclude only the box "ubuntu1604-foreman-1.24". Notice the escaped '.' character to match the specific character instead of any single character
    - "^centos7-fips" # exclude any box that starts with "centos7-fips"
    - "foreman-1\\.(?:[2][0-3])" # exclude any foreman-1.20 to foreman-1.23 version box
```

Reply:
Let's roll with regex for now since you can do simple and complex and see where it takes us.